src/Apps/__tests__/Fixtures/Sale.ts

```diff
@@ -9,3 +10,8 @@ export const SaleAuction: routes_AuctionQueryResponse["sale"] = {
   is_preview: false,
   registrationStatus: null,
 }
+
+export const SaleAuction: AuctionApp_sale = {
+  " $refType": undefined,
+  id: "yuki-contemporary-art-july-22th",
```

Comment:
Something tells me this sort of thing is a convention that i've learned before and forgotten but can you remind me?

Reply:
Is there a reason that this is tied to a staging sale? Should this be a more direct test flag-value, like `an-example-auction-sale`?

Reply:
This is just a fixture so we can name it however. I probably plucked this from another fixture. Agree, that we can rename this to be more example-focused!